tests/files/controller/auth_document.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
As I suggested on Slack, as a proof of concept you should be able to strip everything out of here except the ID and the public key.

Reply:
This file should be renamed.